emummc/source/emuMMC/emummc.h

```diff
@@ -55,15 +56,17 @@ uint64_t sdmmc_wrapper_controller_close(int mmc_id);
 uint64_t sdmmc_wrapper_read(void *buf, uint64_t bufSize, int mmc_id, unsigned int sector, unsigned int num_sectors);
 uint64_t sdmmc_wrapper_write(int mmc_id, unsigned int sector, unsigned int num_sectors, void *buf, uint64_t bufSize);
 
-// TODO: check if FatFS internal buffers are good (perf wise) to have a x16 alignment.
 typedef struct _file_based_ctxt
 {
+	FATFS sd_fs;
 	uint64_t parts;
 	uint64_t part_size;
-	FATFS *sd_fs;
 	FIL fp_boot0;
+	DWORD clmt_boot0[0x400];
```

Comment:
```suggestion
	DWORD clmt_boot0[0x400];
```

```suggestion
	DWORD clmt_boot1[0x400];
```

```suggestion
	DWORD clmt_gpp[0x8000];
```

Would not be better to use a define instead of "magic numbers" here ? (same for the corresponding .c file).

Reply:
@LittleWhite-tb maybe, but this should be brought up on the original sdmmcv2 PR in the emummc repo, as it's not a part of exosphere2.